src/home/home.html

```diff
@@ -6,27 +6,27 @@
       <div class="subtitle">
         An interface for DAO to DAO interactions, such as token swaps, co-liquidity provision, and joint venture formation.
       </div>
-      <div class="buttons">
-        <div class="buttonContainer">
-          <div class="contribute" click.delegate="navigate('deals')">
-            <div class="title">
-              <div class="header">All Deals</div>
-              <div class="arrow">&#8599;</div>
-            </div>
+      <div class="cards">
+        <pcard type="default" width="360px" click.delegate="navigate('deals')">
```

Comment:
why type="default"?  just make that the default in the pcard code.  It's not really a default if you have to specify "default"